index.html

```diff
@@ -14,52 +14,67 @@
-        or website Look <br />through some of my work and <br />experience! If
-        you like what you see and <br />have a project you need coded, don’t
-        <br />hestiate to contact me.
-      </p>
-      <div class="media">
-        <h3>LET'S CONNECT</h3>
-        <div class="socials">
-          <ul>
-            <li class="icon">
-              <a href="https://twitter.com/mwovi_peter"
-                ><img src="./images/twitter.png" alt="twitter link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
-                ><img src="./images/Linkedin_icon.png" alt="linked link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="#"><img src="./images/M.png" alt="M" /></a>
-            </li>
-            <li class="icon">
-              <a href="https://github.com/peter-mwovi/"
-                ><img src="./images/github.png" alt="github link"
-              /></a>
-            </li>
-            <li class="icon">
-              <a href="#"><img src="./images/doits.png" alt="doit" /></a>
-            </li>
-          </ul>
+      <nav>
+        <ul class="nav-links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="flex-container3">
+      <section class="background">
+        <h1>I’m Peter Mwovi. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <div class="media">
+          <h3>LET'S CONNECT</h3>
+          <div class="socials">
+            <ul>
+              <li class="icon">
+                <a href="https://twitter.com/mwovi_peter"
+                  ><img src="./images/twitter.png" alt="twitter link"
+                /></a>
+              </li>
+              <li class="icon">
+                <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
+                  ><img src="./images/Linkedin_icon.png" alt="linked link"
+                /></a>
+              </li>
+              <li class="icon">
+                <a href="#"><img src="./images/M.png" alt="M" /></a>
+              </li>
+              <li class="icon">
+                <a href="https://github.com/peter-mwovi/"
+                  ><img src="./images/github.png" alt="github link"
+                /></a>
+              </li>
+              <li class="icon">
+                <a href="#"><img src="./images/doits.png" alt="doit" /></a>
+              </li>
+            </ul>
+          </div>
         </div>
-      </div>
-    </section>
+      </section>
+    </div>
 
     <div class="grid-container">
```

Comment:
- [optional] please kindly consider using `section tags` to embody your work section and about section. This will help to divide up your web page into standalone sections of related content. Using the section element over a generic container element like div can help make your code more accessible and understandable to search engines, browsers, assistive technologies, and other developers.

Reply:
please this is just an additional comment for the confusion since i can see another reviewer just required changes while i had already approved. please the student implemented the changes. you can see from this vid below.
![demo33](https://github.com/peter-mwovi/portfolio_setup/assets/65298048/b1f3ffee-dd6d-4ae2-956e-31e90509dd51)